### PR TITLE
Ported tracker bio avatar to use avatar helper and cleaned up css

### PR DESCRIPTION
**Pull request by chrisnojima**
@keybase/react-hackers This uses the shared avatar component for the avatar in the tracker
@gabriel i changed the css a little bit, we should discuss. In general we should design our css so that when we are resizing various elements the logical layout will still 'hold'. For example the grey/green follow/following circles on the avatar were using position absolute with a top/left. But really they're bottom /right aligned so it's better to define them that way. Then if we make the avatar larger they'll still 'work' instead of being in the middle of the avatar.  I also moved the merging of some props out of the component and into the css. I think this is a little cleaner of a pattern.

### CODE REVIEW

File: shared/styles/style-guide.desktop.js [hunk 1]
```diff
@@ -44,6 +44,7 @@ export const globalColorsDZ2 = {
 
   lightGrey: '#ebebeb',
   lightGrey2: '#f6f6f6',
+  lightGrey3: '#e0e0e0',
```
Comment (chrisnojima) [inline shared/styles/style-guide.desktop.js]: updated in the style guide

File: shared/tracker/bio.render.desktop.js [hunk 2]
```diff
@@ -76,13 +75,11 @@ export default class BioRender extends Component {
...
           <div style={styles.avatarOuter}>
-            <Paper onClick={() => this.onClickAvatar()} style={styles.avatarContainer} zDepth={1} circle>
-              <img src={(userInfo.avatar) || noAvatar} style={styles.avatar}/>
-            </Paper>
+            <Avatar onClick={() => this.onClickAvatar()} url={userInfo.avatar} size={75} />
             {(followsYou || currentlyFollowing) &&
               <div>
-                <div style={followsYou ? {...followBadgeStyles.followBadge, ...followBadgeStyles.followsYou} : {...followBadgeStyles.followBadge, ...followBadgeStyles.notFollowsYou}} />
-                <div style={currentlyFollowing ? {...followBadgeStyles.followBadge, ...followBadgeStyles.following} : {...followBadgeStyles.followBadge, ...followBadgeStyles.notFollowing}} />
+                <div style={followsYou ? followBadgeStyles.followsYou : followBadgeStyles.notFollowsYou} />
```
Comment (chrisnojima) [inline shared/tracker/bio.render.desktop.js]: let's move this kind of logic into the style itself

File: shared/tracker/bio.render.desktop.js [hunk 3]
```diff
@@ -255,34 +245,42 @@ const styles2 = {
   }
 }
 
+const followBadgeCommon = {
```
Comment (chrisnojima) [inline shared/tracker/bio.render.desktop.js]: this stuff is shared so let's treat it as such

File: shared/tracker/bio.render.desktop.js [hunk 4]
```diff
@@ -255,34 +245,42 @@ const styles2 = {
...
+
 const followBadgeStyles = {
-  followBadge: {
-    position: 'absolute',
-    background: '#fff',
-    width: 14,
-    height: 14,
-    borderRadius: '50%',
-    border: '2px solid #fff',
-    boxShadow: '0px 3px 3px rgba(0,0,0,0.1)'
```
Comment (chrisnojima) [inline shared/tracker/bio.render.desktop.js]: not sure why we have this dropshadow. its not in the designs

**Comment by MarcoPolo** [comment]
:ship: :it:

**Pull request merged**